torch/nn/modules/module.py

```diff
@@ -349,7 +349,7 @@ def parameters(self, memo=None):
             <class 'torch.FloatTensor'> (20L,)
             <class 'torch.FloatTensor'> (20L, 1L, 5L, 5L)
         """
-        for name, param in self.named_parameters():
+        for name, param in self.named_parameters(memo):
```

Comment:
Thanks for spotting this, but it appears to me that it's unnecessary and `memo` could be removed from `parameters`. Once `named_parameters` is called it will correctly pass its `memo` around and will never call `parameters()` again.